plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/PolylineAnnotation.kt

```diff
@@ -389,20 +391,41 @@ class PolylineAnnotation(
     moveDistancesObject: MoveDistancesObject
   ): LineString? {
     val points = geometry.coordinates()
-      .map {
-        mapCameraManagerDelegate.pixelForCoordinate(it)
```

Comment:
I wonder if `pixelForCoordinate` should correctly reproject points even with 3d enabled? Will the users have the same issues with `pixelForCoordinate`?

Reply:
Don't we want to introduce some helper API to shift List<Point> with 3d enabled?

Reply:
`mapCameraManagerDelegate.pixelForCoordinate` doesn't work well with 3d, this is confirmed from gl-native side.

Reply:
> `mapCameraManagerDelegate.pixelForCoordinate` doesn't work well with 3d, this is confirmed from gl-native side.

Got it. How about adding some util that moves list of points with 3d enabled though? Reuse it in both Polygon* classes and make public in case users somehow needs to shift points themselves?

Reply:
Sounds good. Will do.